dev-python/XenAPI/XenAPI-2.14.ebuild

```diff
@@ -1,8 +1,8 @@
-# Copyright 1999-2014 Gentoo Foundation
+# Copyright 1999-2019 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=5
-PYTHON_COMPAT=( python2_7 )
+EAPI=7
+PYTHON_COMPAT=( python2_7 python3_{5,6,7,8} )
```

Comment:
Please drop py2_7 and py3_5. Both will be going away.

Reply:
Why? What if someone will keep py2 locally? Keeping them in the ebuild does not block Gentoo to drop 2.7/3.5 in the future.

Reply:
Yes, it absolutely blocks us from removing py2 in the future. This process has already begun. If the individual wants to use py2 they can do so in an overlay.

Reply:
I think having a PYTHON_COMPAT in ebuilds does not prevent from dropping python2.7 because it will be dropped from PYTHON_TARGETS and thus result in a no-op. It just declares it can work with python2.7.

But ok, I've removed 2.7/3.5 and updated the PR.